doc/source/rest.j2

```diff
@@ -838,6 +838,17 @@ Operations between metrics can also be done, such as:
 
 {{ scenarios['get-aggregates-between-metrics']['doc'] }}
 
+List top N instances for that consume the most cpu last hour
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+{{ scenarios['use-case1-top-cpuutil-per-instances']['doc'] }}
+
+The following is configured so that `stop` - `start` = `granularity` will get
+only one point per instance.
+
+This will give all information needed to order by cpu.util timeseries which can
+be filtered down to N results..
```

Comment:
s/.././

we can maybe move the example REST below the text but it's not important.